src/components/map/index.tsx

```diff
@@ -99,7 +99,7 @@ function Map({ initialLocation, shops, onViewportChange }: Props): JSX.Element {
             onViewportChange={handleViewportChange}
             mapboxApiAccessToken={process.env.mapboxToken}
             onError={console.info}
-            mapStyle="mapbox://styles/chrstnst/ck6zhy4iv4avr1is7blvy0ng9"
+            mapStyle="mapbox://styles/chrstnst/ck9q6dr2w1f1k1jufx5csfolz"
```

Comment:
nit: maybe we can add this as a envvar, wdyt?

Reply:
Hm, this will honestly not really change that often or depend on an environment. I’ll leave it hardcoded for the time being, there’s no need to make this variable